### PR TITLE
Add materialized view support for BigQuery

### PR DESCRIPTION
**Pull request by ryanyuan**
---
Add materialized view support for BigQuery Hooks/Operators.
https://cloud.google.com/bigquery/docs/materialized-views-intro

### CODE REVIEW

**Comment by ryanyuan** [comment]
PTAL @turbaszek

File: airflow/providers/google/cloud/operators/bigquery.py [hunk 1]
```diff
@@ -875,6 +877,7 @@ class BigQueryCreateEmptyTableOperator(BaseOperator):
         'gcs_schema_object',
         'labels',
         'view',
+        'materialized_view',
         'impersonation_chain',
     )
     template_fields_renderers = {"table_resource": "json"}
```
Comment (turbaszek) [inline airflow/providers/google/cloud/operators/bigquery.py]:
```suggestion
    template_fields_renderers = {"table_resource": "json", "materialized_view": "json"}
```

**Review by turbaszek — APPROVED** [review]
Looks good to me 🚀 Only one suggestion.

**Comment by ryanyuan** [comment]
> Looks good to me 🚀 Only one suggestion.

Done. Thanks @turbaszek

**Review by turbaszek — APPROVED** [review]

**Pull request merged**